src/routes/dashboard/fragment-wizard/create-fragment-modal.tsx

```diff
@@ -68,6 +83,10 @@ export const CreateFragmentModal = (props: CreateFragmentModalProps) => {
 		navigate(`/edit/${generatedFragment.id}`);
 	};
 
+	const handleChange = (x: CreateOptions) => {
+		setSelectedCreateOption(parseInt(CreateOptions[x]));
```

Comment:
```suggestion
		setSelectedCreateOption(x);
```

Instead of having to parse, let's just change the value to be the enum (number) 🤔